tests/run-multitests.py

```diff
@@ -3,22 +3,36 @@
 # This file is part of the MicroPython project, http://micropython.org/
 # The MIT License (MIT)
 # Copyright (c) 2020 Damien P. George
+#
+# run-multitests.py
+# Runs a test suite that relies on two micropython instances/devices
+# interacting in some way. Typically used to test networking / bluetooth etc.
+
 
 import sys, os, time, re, select
 import argparse
 import itertools
 import subprocess
 import tempfile
 
-sys.path.append("../tools")
+test_dir = os.path.abspath(os.path.dirname(__file__))
+
+if os.path.abspath(sys.path[0]) == test_dir:
+    # remove the micropython/tests dir from path to avoid
+    # accidentally importing tests like micropython/const.py
+    sys.path.pop(0)
+
+sys.path.insert(0, f"{test_dir}/../tools")
```

Comment:
No other Python tool/test scripts in this repo use f-strings (yet, except mpremote), and since this is a very simple case please just do `test_dir + "/../tools"`.

Also 2 cases below.

Reply:
No worries, they've just become habit for me now :-)
That's fixed now thanks